### DISCUSSION TITLE
How to update all dependencies except Openssl

### DISCUSSION THREAD

**Discussion by jvasanthan** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#2003] [Q&A]
Hi Team, 
How can I update all other dependencies to latest version except openssl. I want to keep openssl to 1.1.1g. 
Please let me know

**Comment by sirknightj** [comment]
Hi,

You can find the CMake files where the dependencies are imported in the Dependencies folder:

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/tree/master/CMake/Dependencies

From there, look for the [`ExternalProject_Add`](https://cmake.org/cmake/help/latest/module/ExternalProject.html) tag and change the `GIT_TAG` to a more recent commit.
